tests/test_tracing.py

```diff
@@ -732,6 +734,459 @@ def _basic_common_checks(
         self.assertEqual(span.get_tag("request_id"), "abc123")
 
 
+class TestServiceMapping(unittest.TestCase):
```

Comment:
Really good job on this battery of tests. Fall-through config is important but also behavior is often left untested and results in latent bugs months later, and these seem very thorough.

Reply:
thank you! `Fall-through config is important` not sure if you are implying I should add tests?